packages/amplication-server/src/core/resource/dto/ResourceCreateInput.ts

```diff
@@ -18,6 +18,11 @@ export class ResourceCreateInput {
   })
   description!: string;
 
+  @Field(() => Boolean, {
+    nullable: true,
+  })
+  requireAuthenticationEntity?: boolean;
+
```

Comment:
If this is optional, don't we want a default value? What will happen if we don't send anything?